tests/host_tools/cargo_build.py

```diff
@@ -57,43 +60,27 @@ def cargo_test(path, extra_args='', extra_env=''):
     run(cmd, shell=True, check=True)
 
 
-def get_firecracker_binaries(root_path, features=''):
+def get_firecracker_binaries(features=''):
     """Build the Firecracker and Jailer binaries if they don't exist.
 
     Returns the location of the firecracker related binaries eventually after
     building them in case they do not exist at the specified root_path.
     """
-    extra_args = \
-        '--release --target {} >/dev/null 2>&1'.format(DEFAULT_BUILD_TARGET)
-    extra_env = 'TARGET_CC=musl-gcc'
-
-    if features == '':
-        cargo_binaries_rel_path = CARGO_RELEASE_REL_PATH
-    else:
-        raise UnknownFeatureException
-
-    path_to_binaries = os.path.join(
-        root_path,
-        os.path.join(
-            cargo_binaries_rel_path,
-            RELEASE_BINARIES_REL_PATH
-        )
+    if features != '':
+        raise UnknownFeatureException()
+
+    target = DEFAULT_BUILD_TARGET
+    cd_cmd = "cd {}".format(FC_WORKSPACE_DIR)
```

Comment:
Isn't cargo able to do the build in another directory? I was thinking that we can use that option instead of changing the directory.

Reply:
The idea here was to use the default target directory (set by our `.cargo/config`), so that the cargo build cache/artifacts can be reused.

Also, the current dir change only affects the spawned subprocess that is used to run `cargo`.